### PR TITLE
Scribus 1.5.4

### PR DESCRIPTION
**Pull request by jurf**
(This obsoletes #333, trying to import the git history is a PITA, and I converted it to YAML anyway so it’s not worth it anymore.)

This still builds from master, but I’ll add it here anyway, since:

 * It’ll prevent work duplication (I invested a tonne of time into this)
 * Flathub builds all 4 platforms
 * Feedback is useful

I’ll keep the development over at [jurf/scribus-flatpak](https://github.com/jurf/scribus-flatpak), though I’ll merge the changes here from time to time.

### CODE REVIEW

**Comment by jurf** [comment]
bot, build net.scribus.Scribus

**Comment by jurf** [comment]
bot, build net.scribus.Scribus

**Comment by jurf** [comment]
bot, build net.scribus.Scribus

**Comment by jurf** [comment]
bot, build net.scribus.Scribus

**Comment by jurf** [comment]
bot, build net.scribus.Scribus

**Comment by jurf** [comment]
It has actually built, I cannot believe it.

File: net.scribus.Scribus.yml [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+runtime:             org.kde.Platform
+runtime-version:     '5.10'
+sdk:                 org.kde.Sdk
+
+command:             scribus
+rename-desktop-file: scribus.desktop
+rename-appdata-file: scribus.appdata.xml
+rename-icon:         scribus
+
+# Mark as nightly
```
[the remark this answers is not in the document]
Comment (jurf) [reply net.scribus.Scribus.yml]: Yeah, but I’ll keep it to simplify the sync until 1.6 arrives, which will be quite some time.

File: net.scribus.Scribus.yml [hunk 2]
```diff
@@ -0,0 +1,338 @@
+app-id:              net.scribus.Scribus
+
+runtime:             org.kde.Platform
+runtime-version:     '5.10'
```
[the remark this answers is not in the document]
Comment (jurf) [reply net.scribus.Scribus.yml]: TIL.

File: net.scribus.Scribus.yml [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+rename-appdata-file: scribus.appdata.xml
+rename-icon:         scribus
+
+# Mark as nightly
+tags:
+  - nightly
+desktop-file-name-prefix: '(Nightly) '
+
+finish-args:
+  # X11 + XShm access
```
[the remark this answers is not in the document]
Comment (jurf) [reply net.scribus.Scribus.yml]: Isn’t Qt Wayland considered not-production-ready-yet?

**Comment by jurf** [comment]
@barthalion thanks for the review, I’ll push the changes tomorrow.

File: net.scribus.Scribus.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jurf) [reply net.scribus.Scribus.yml]: Actually scratch that, I had to change the scribus source anyway.

**Comment by jurf** [comment]
bot, build net.scribus.Scribus

**Comment by jurf** [comment]
bot, build net.scribus.Scribus

**Comment by jurf** [comment]
bot, build net.scribus.Scribus

**Comment by jurf** [comment]
bot, build net.scribus.Scribus

**Comment by jurf** [comment]
bot, build net.scribus.Scribus

**Comment by jurf** [comment]
OK, **finally** builds everywhere.

So, what now? Arch already [uses](https://www.archlinux.org/packages/community/x86_64/scribus/) 1.5.4 in the stable repositories. Personally, I would really like to see this on Flathub (not just because I invested a _tonne_ of time into it). Maybe change the label to Beta or something, to show it’s not completely stable? See also my comment in #501.

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/net.scribus.Scribus

**Comment by nedrichards** [comment]
@jurf anyone else we should add to the repo?

**Comment by jurf** [comment]
Wait… this actually happened.

O.O

Thanks.

@nedrichards I’ll ask on IRC.

**Pull request closed without merging**